source/code-snippets/find.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func main() {
+	// Replace the uri string with your MongoDB deployment's connection string.
+	uri := "mongodb+srv://<username>:<password>@<cluster-address>/test?w=majority"
+
+	ctx := context.TODO()
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(uri))
+
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err = client.Disconnect(ctx); err != nil {
+			panic(err)
+		}
+	}()
+
+	// begin find
+	myCollection := client.Database("sample_training").Collection("zips")
+	filter := bson.D{{"pop", bson.D{{"$lte", 500}}}}
+
+	cur, err := myCollection.Find(ctx, filter)
+	// end find
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var results []bson.M
+	if err = cur.All(context.TODO(), &results); err != nil {
```

Comment:
```suggestion
	if err = cur.All(ctx, &results); err != nil {
```

Question: Is there a reason a new context was created here?

Reply:
Updating all context statements to `context.TODO()` per https://github.com/mongodb/docs-golang/pull/5#discussion_r676969118.